G1ANT.Addon.MSOffice/G1ANT.Addon.MSOffice/Commands/OutlookMoveToCommand.md

```diff
@@ -0,0 +1,40 @@
+| ------------------- | ------------------------------------------------------------ | -------- | ------------------------------------------------------------ | ------------------------------------------------------------ |
+| `item`              | [outlookmail](../../../g1ant.language/structures/outlookmailstructure.md) or [outlookfolder](../../../g1ant.language/structures/outlookfolderstructure.md) | yes      |                                                              | An item (a message or a folder) to be moved                  |
+| `destinationfolder` | [outlookfolder](../../../g1ant.language/structures/outlookfolderstructure.md) | yes      |                                                              | Destination Outlook folder                                   |
+| `if`           | [bool](../../../g1ant.language/structures/booleanstructure.md) | no       | true                                                        | Executes the command only if a specified condition is true   |
+| `timeout`      | [timespan](../../../g1ant.language/structures/timespanstructure.md) | no       | [♥timeoutcommand](../../../appendices/special-variables.md) | Specifies time in milliseconds for G1ANT.Robot to wait for the command to be executed |
+| `errorcall`    | [procedure](../../../g1ant.language/structures/procedurestructure.md) | no       |                                                             | Name of a procedure to call when the command throws an exception or when a given `timeout` expires |
+| `errorjump`    | [label](../../../g1ant.language/structures/labelstructure.md) | no       |                                                             | Name of the label to jump to when the command throws an exception or when a given `timeout` expires |
+| `errormessage` | [text](../../../g1ant.language/structures/textstructure.md)  | no       |                                                             | A message that will be shown in case the command throws an exception or when a given `timeout` expires, and no `errorjump` argument is specified |
+| `errorresult`  | [error](../../../g1ant.language/structures/errorstructure.md) | no       |                                                             | Name of a variable that will store the returned exception    |
+
+For more information about `if`, `timeout`, `errorcall`, `errorjump`, `errormessage` and `errorresult` arguments, see [Common Arguments](../../../appendices/common-arguments.md) page.
+
+## Example
+
+In the example below, Outlook opens silently in the background, its source and destination folders are retrieved to variables with the `outlook.getfolder` commands, then the first email message from the Inbox folder is moved to the destination folder:
+
+```G1ANT
+♥outlookInboxFolder = john.doe@g1ant.com\Inbox
+♥outlookMoveToFolder = john.doe@g1ant.com\[Gmail]\New
```

Comment:
what is this: "[Gmail]"? If I don't know it, maybe a little description here will be helpful for others :)

Reply:
I don't see a reminder anywhere but you leave it as you like.

Reply:
OK, the reminder was in another file :D